conf/routes

```diff
@@ -136,6 +136,9 @@ GET    /v3/scheduleplans/:guid     @org.sagebionetworks.bridge.play.controllers.
 POST   /v3/scheduleplans/:guid     @org.sagebionetworks.bridge.play.controllers.SchedulePlanController.updateSchedulePlan(guid: String)
 DELETE /v3/scheduleplans/:guid     @org.sagebionetworks.bridge.play.controllers.SchedulePlanController.deleteSchedulePlan(guid: String)
 
+# Bridge Exporter
+POST /v3/recordExportStatuses      @org.sagebionetworks.bridge.play.controllers.HealthDataController.setRecordsStatus
```

Comment:
Please follow the convention for upload statuses and use POST /v3/exportstatuses (all lowercase)